src/main/java/org/mockbukkit/mockbukkit/simulate/entity/PlayerSimulation.java

```diff
@@ -104,12 +104,12 @@ public PlayerSimulation(PlayerMock playerMock)
 	/**
 	 * Simulates the player damaging a block. Note that this method does not anything unless the player is in survival
 	 * mode. If {@code InstaBreak} is set to true by an event handler, a {@link BlockBreakEvent} is immediately fired.
-	 * The result will then still be whether or not the {@link BlockDamageEvent} was cancelled or not, not the later
+	 * The result will then still be whether the {@link BlockDamageEvent} was cancelled or not, not the later
 	 * {@link BlockBreakEvent}.
 	 *
 	 * @param block The block to damage.
 	 * @return the event that was fired, {@code null} if the player was not in
-	 * survival gamemode.
+	 * survival game mode.
```

Comment:
I don't think this is a good change. Gamemode is an established term that I don't think needs to be broken up

Reply:
Okay, there's defnitely inconsistencies here in that case, because the bukkit api defines gamemode as the enum `org.bukkit.GameMode`, which implies that it is two separate words